WordPress/src/main/res/layout/subfilter_list_item.xml

```diff
@@ -3,7 +3,8 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     style="@style/SubfilterSiteTagItem"
-    android:layout_width="match_parent">
+    android:layout_width="match_parent"
+    android:focusable="true">
```

Comment:
What was the reason for adding `focusable` here? Let me know. Thanks 😄

Reply:
Oh you are right 😊! I initially was misusing it thinking I needed to have it there to announce the whole group and forgot to remove 🤦‍♂️. Thanks for catching, changed [here](7b2adcbbb78dddbfe3cf44092f4ef11c57631bad)